### PR TITLE
Webpack bundle analyzer

### PR DESCRIPTION
**Pull request by gretzky**
Tossed in [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer) and corresponding cli commands to start it up in either dev or prod. This tool is pretty awesome, and allows you to view in localhost without having to upload. I left the regular analyze script in tact.

Not sure if this is something that could/should be pulled in now, or for a future release. Open to either!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
   "scripts": {
     "analyze:clean": "rimraf stats.json",
     "preanalyze": "npm run analyze:clean",
+    "analyze:watch": "cross-env NODE_ENV=development ANALYZE=true node server",
+    "analyze:watchProd": "cross-env NODE_ENV=production ANALYZE=true node server",
```
Comment (avdeev) [inline package.json]: I don't very like commans with enviromens

Can we pass NODE_ENV from cli?

Comment (gretzky) [reply package.json]: I'm open to anything!

Comment (avdeev) [reply package.json]: @gretzky I think, we can use `ANALYZE=true yarn start` and `ANALYZE=true yarn start:prod`

And we need advanced-configuration section, such as CRA

https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#advanced-configuration

**Comment by blling** [comment]
@gretzky RBP always has it\`s `analyze`, how should their respective responsibilities be divided? Or allow them to repeat?

**Comment by gretzky** [comment]
@blling This PR pulls in the bundle analyzer that loads in the browser, so the responsibilities are really just divided over wanting a hard copy (`stats.json`) of your bundle, or just being able to look at it while developing / for quick insight

**Comment by blling** [comment]
>This PR pulls in the bundle analyzer that loads in the browser, so the responsibilities are really just divided over wanting a hard copy (stats.json) of your bundle, or just being able to look at it while developing / for quick insight

Get it, i\`m sorry, my stupid.

**Comment by gretzky** [comment]
I'm going to close this for now in favor of reopening with some changes inspired by @avdeev

**Pull request closed without merging**